test/cli.js

```diff
@@ -43,7 +46,7 @@ function execCli(args, opts, cb) {
 	var stderr;
 
 	var processPromise = new Promise(function (resolve) {
-		child = childProcess.spawn(process.execPath, [path.relative(dirname, cliPath)].concat(arrify(args)), {
+		child = childProcess.spawn(process.execPath, [path.relative(dirname, cliPath)].concat(args, '--color'), {
```

Comment:
We need to turn on colors, since `Istanbul` [disables colors at this condition](https://github.com/chalk/supports-color/blob/master/index.js#L41)